lib/stink_bomb/configuration.rb

```diff
@@ -7,21 +7,11 @@ def initialize(options = {})
     end
 
     def raise=(error_class)
-      remove_bomb(RaiseBomb)
-      if error_class == true
-        add_bomb(RaiseBomb.new)
-      elsif error_class
-        add_bomb(RaiseBomb.new(error_class: error_class))
-      end
+      add_bomb(RaiseBomb, error_class)
```

Comment:
the deduping make sense, but can the function be named differently so that its more obvious what is going on in the function?  i dont have great ideas though... maybe `create_bomb_singleton`